custom_components/neviweb130/schema.py

```diff
@@ -293,14 +294,14 @@
 SET_SETPOINT_MAX_SCHEMA = vol.Schema(
     {
         vol.Required(ATTR_ENTITY_ID): cv.entity_id,
-        vol.Required(ATTR_ROOM_SETPOINT_MAX): vol.All(vol.Coerce(float), vol.Range(min=8, max=36)),
+        vol.Required(ATTR_ROOM_SETPOINT_MAX): vol.All(vol.Coerce(float), vol.Range(min=11, max=30)),
```

Comment:
@claudegel, do you know where those initial values were from? If another device can go as low as 8oC for the minimum, I can leave it to 8 and 36 for max but then raise an error if we're having unsupported values for the TH6500WF. What do you think?